FreeRTOS/FreeRTOS-Plus-TCP
# Remove dependency between mps2 network interface and demo

Remove dependency between mps2 network interface and demo

Description
-----------
copy cmsis directory from the demo, to the network interface making not rely on external dependencies.

Test Steps
-----------
demo compiles normally 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
